bigchaindb/core.py

```diff
@@ -597,13 +606,14 @@ def get_unvoted_blocks(self):
         unvoted = r.table('bigchain') \
             .filter(lambda block: r.table('votes').get_all([block['id'], self.me], index='block_and_voter')
                     .is_empty()) \
-            .order_by(r.desc('block_number')) \
+            .order_by(r.desc(r.row['block']['timestamp'])) \
```

Comment:
We should probably use `r.asc` to retrieve a list starting from the oldest block. This way the oldest blocks are voted first.